tests/backend/test_design_step_bbox.py

```
@@ -0,0 +1,99 @@
+"""Integration tests for the design STEP export endpoint."""
+
+from __future__ import annotations
+
+from pathlib import Path
+from typing import Dict, Tuple
```

Ruff (0.14.1)

`typing.Dict` is deprecated, use `dict` instead

(UP035)